packages/inventory/src/containers/ServicePointProfile/tests/index.test.tsx

```diff
@@ -84,7 +84,7 @@ describe('Profile view Page', () => {
   });
 
   it('renders correctly', async () => {
```

Comment:
Shouldn't we be checking for UI elements in this test case? It appears the description should change to `renders when data is absent` since the mocked fetch responses are empty